packages/server/src/integrations/base/sql.ts

```diff
@@ -8,9 +8,24 @@ import {
   Operation,
   RelationshipsJson,
 } from "../../definitions/datasource"
+import { isIsoDateString } from "../utils"
 
 type KnexQuery = Knex.QueryBuilder | Knex
 
+function parseBody(body: any) {
+  for (let [key, value] of Object.entries(body)) {
+    if (typeof value !== "string") {
+      continue
+    }
+    if (isIsoDateString(value)) {
```

Comment:
I was hoping this day wouldn't come so soon! :laughing:

But we will likely need to account for a bunch of different types as they pop up.

Reply:
Yeah, I wasn't sure how else to really handle this, other than to try work out what the types are, its not a perfect fix but I think its better we just convert things into their native types and let knex do the heavy lifting in terms of what various SQL DBs expect in their inputs!

Luckily at least our forms library outputs very static things for these types, e.g. dates and times are always ISO format so its very easy to detect!